maintainers/maintainer-list.nix

```diff
@@ -17014,6 +17014,12 @@
     githubId = 1153271;
     name = "Sander van der Burg";
   };
+  Sanskarzz = {
```

Comment:
The commit adding those lines should be named ``[maintainers: add Sanskarzz``

Reply:
It shouldn't include the leading `[`, should it, @SuperSandro2000?